### ISSUE TITLE
Checkov doesn't seem to support variables for aws_ebs_volume / encrypted

### ISSUE THREAD

**Issue by james-gonzalez** [bridgecrewio/checkov#81]
**Describe the bug**
Checkov doesn't seem to support variables for aws_ebs_volume / encrypted. 

Perhaps this was by design, but please explain if otherwise.

i.e.
```
resource "aws_ebs_volume" "default" {
  availability_zone = var.volume_az
  size                     = var.volume_size
  type                    = var.volume_type
  encrypted          = var.volume_encryption
}

variable "volume_encryption" {
  type        = bool
  description = "Volume encryption enabled or disabled."
  default     = true
}
```
**To Reproduce**
1. Run `checkov -d ./terraform-code.tf`
2. You get: 
```
Check: CKV_AWS_3: "Ensure all data stored in the EBS is securely encrypted "
	FAILED for resource: aws_ebs_volume.default
```

**Expected behavior**
It should interpret the variable and pass. As it's set to `true` by default and nothing is overriding that.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Checkov Version `checkov==1.0.102`

**Comment by tronxd** [comment]
Fixed in #82 .

**Comment by schosterbarak** [comment]
@james-gonzalez  thank you for reporting this! do you mind testing again with latest version?
